### PR TITLE
Create Exporter.yml

### PR DESCRIPTION
**Pull request by ignatandrei**
Up for grabs for Exporter : http://ignatandrei.github.io/Exporter/

### CODE REVIEW

File: _data/projects/Exporter.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ - PDF
+ - ODS
+ - ODT
+ - JSON
+ - List
+ - DataTable
+ - IDataReader
+ - CSV
+ - RSS
+ -OPML
```
Comment (shiftkey) [inline _data/projects/Exporter.yml]: I think the YAML parser is having trouble with this line

Comment (ignatandrei) [inline _data/projects/Exporter.yml]: modified now

File: _data/projects/Exporter.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+upforgrabs:
+  name: helpwanted
```
Comment (shiftkey) [inline _data/projects/Exporter.yml:19]: this tag was fine as "help wanted"

File: _data/projects/Exporter.yml (hunk 1 above)
Comment (shiftkey) [inline _data/projects/Exporter.yml]: Still seems to be failing:

```
$ ruby scripts/cibuild.rb
225 files processed - 1 errors found:
/home/travis/build/up-for-grabs/up-for-grabs.net/_data/projects/Exporter.yml - Unable to parse the contents of file - Line: 17, Offset: 0, Problem: could not find expected ':'
```

File: _data/projects/Exporter.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ - ODT
+ - JSON
+ - List
+ - DataTable
+ - IDataReader
+ - CSV
+ - RSS
+ - OPML
+upforgrabs:
+  name: helpwanted
```
Comment (shiftkey) [inline _data/projects/Exporter.yml:19]: this tag was fine as `"help wanted"`

**Comment by shiftkey** [comment]
Closing this out as inactive. Feel free to open a fresh PR if you'd still like to see your project listed.

**Pull request closed without merging**